app/models/katello/host/content_facet.rb

```diff
@@ -32,10 +32,10 @@ class ContentFacet < Katello::Model
       validates_with Validators::ContentViewEnvironmentValidator
 
       def update_repositories_by_paths(paths)
-        prefixes = %w(/pulp/deb/ /pulp/repos/ /pulp/content/)
+        prefixes = %w(/pulp/content/)
```

Comment:
we still want to support clients fetching content on the old paths, while advertising the new paths, so this needs to stay

Reply:
Thank you!